trigger/src/main/scala/core/models/tables/JdbcTypeMapper.scala

```diff
@@ -0,0 +1,56 @@
+package core.models.tables
+
+import core.models.enums.{EventTypes, JobStatuses, JobTypes}
+import core.models.enums.EventTypes.EventType
+import core.models.enums.JobStatuses.JobStatus
+import core.models.enums.JobTypes.JobType
+import play.api.libs.json.{JsValue, Json}
+import slick.ast.BaseTypedType
+import slick.jdbc.JdbcType
+import scala.util.Try
+import slick.jdbc.MySQLProfile.api._
+
+object JdbcTypeMapper {
+
+  implicit val eventTypeMapper: JdbcType[EventType] with BaseTypedType[EventType] =
```

Comment:
isn't `TypedType` a bit redundant?

Reply:
Yes it is. It was autofilled by editor. Removed